src/types/react-lottie.d.ts

```diff
@@ -0,0 +1,28 @@
+declare module 'react-lottie' {
```

Comment:
https://zippy.gfycat.com/TastyBestGermanpinscher.webm

Lets upstream this!

Reply:
yas!